mbuild/compound.py

```diff
@@ -69,53 +69,64 @@ def load(filename, relative_to_module=None, compound=None, coords_only=False,
     compound : mb.Compound
 
     """
+
+    if compound is None:
+        compound = Compound()
+
+    # First check if we are loading from an existing parmed or trajectory structure
+    if isinstance(filename_or_topology, pmd.Structure):
+        compound.from_parmed(filename_or_topology,coords_only=coords_only)
```

Comment:
Should we have `**kwargs` here (and three lines below)?

Reply:
I added the `**kwargs` for these conversion methods.